File: Selenium/Activities/Activity2.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
	
	public static void main(String[]args) {
		
		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/login-form");
		driver.getTitle();
		System.out.println("Home page title: " + driver.getTitle());
		
		driver.findElement(By.id("username")).sendKeys("admin");
		System.out.println("Username entered");
		
		driver.findElement(By.id("password")).sendKeys("password");
		System.out.println("Password entered");
		
		driver.findElement(By.xpath("/html/body/div[2]/div/div[2]/div[2]/div/div/button")).click();
		
		System.out.println("Logged in successfully");
		
		//driver.close();
		
	}

}

File: Selenium/Activities/activity2.py
from selenium import webdriver
from selenium.webdriver.firefox.service import service as FirefoxService
from selenium.webdriver.common.by import By
import time

with webdriver.Firefox() as driver:
    #Open URL
    driver.get("https://v1.training-support.net/selenium/login-form")
    print("Page Opened: ", driver.title)

    username = driver.find_element(By.ID, "username").send_keys("admin")
    print("Username Entered")

    password = driver.find_element(By.ID, "password").send_keys("password")
    print("Password Entered")

    login = driver.find_element(By.XPATH, "/html/body/div[2]/div/div[2]/div[2]/div/div/button").click()
    time.sleep(5)
    print("Login Successful")
